lib/actions/preload.coffee

```diff
@@ -50,9 +50,11 @@ getApplicationsWithSuccessfulBuilds = (resin, deviceType) ->
 				build.status == 'success'
 		applications
 
-selectApplication = (expectedError, resin, form, deviceType) ->
+selectApplication = (expectedError, resin, form, deviceType, applicationInfoSpinner) ->
```

Comment:
@pimterry I'd need to `require('resin-cli-visuals')` in that function if I didn't pass the spinner.
Not sure what's the best option.